docs/docs-ref-autogen/visio/toc.yml

```diff
@@ -73,3 +81,229 @@ items:
             uid: 'visio!Visio.ShapeMouseLeaveEventArgs:interface'
           - name: ShapeView
             uid: 'visio!Visio.ShapeView:class'
+      - name: Word
+        href: /javascript/api/word
+      - name: Common APIs
```

Comment:
I didn't think that Common APIs apply for Visio...?

Reply:
They don't. I must have circumvented that check in this version.